tests/api2/jail.py

```diff
@@ -30,20 +30,45 @@ def test_02_verify_iocage_pool():
     assert results.json() == IOCAGE_POOL, results.text
 
 
-def test_03_verify_list_resources_endpoint():
+def test_03_verify_list_resources_is_and_integer():
     results = POST('/jail/list_resource/', {'resource': 'RELEASE'})
     assert results.status_code == 200, results.text
-    assert isinstance(results.json(), list), results.text
+    assert isinstance(results.json(), int), results.text
 
 
-def test_04_fetch_bsd_release():
-    global JOB_ID, RELEASE
-    releases = POST('/jail/list_resource/', {
+def test_04_get_bsd_available_release():
+    global JOB_ID
+    payload = {
         'resource': 'RELEASE',
         'remote': True
-    }).json()
+    }
+    results = POST('/jail/list_resource/', payload)
+    JOB_ID = results.json()
+
+
+def test_05_verify_available_bsd_release():
+    global freeze, RELEASE
+    freeze = False
+    global freeze_msg
+    stop_time = time.time() + 600
+    while True:
+        get_job = GET(f'/core/get_jobs/?id={JOB_ID}')
+        job_status = get_job.json()[0]
+        if job_status['state'] in ('RUNNING', 'WAITING'):
```

Comment:
Not necessary but do you think it might be better to have waiting for job as a common method somewhere and use it ? So if in future if any changes are introduced, we wouldn't be changing this in every tests ;)